internal/kafka/internal/api/admin/private/api/openapi.yaml

```diff
@@ -95,7 +95,7 @@ paths:
           Search criteria.
 
           The syntax of this parameter is similar to the syntax of the `where` clause of an
-          SQL statement. Allowed fields in the search are `cloud_provider`, `name`, `owner`, `region`, and `status`. Allowed comparators are `<>`, `=`, `LIKE`, or `ILIKE`.
+          SQL statement. Allowed fields in the search are `cloud_provider`, `name`, `owner`, `region`, `status` and `cluster_id`. Allowed comparators are `<>`, `=`, `LIKE`, or `ILIKE`.
```

Comment:
Not related to this PR, but I think it would be class to allow the `IN` operator as well - to do things like cluster_id IN ('cluster-id-1', 'cluster-id-2').

@ziccardi @pb82 what do you think?

Reply:
That would be useful for example if you have more than one Enterprise cluster. I'll have a look if that can be supported.

Reply:
Agree. Supporting it should be pretty easy as well.

Reply:
Thanks. 

> Agree. Supporting it should be pretty easy as well.

I'l be happy to review a followup PR my fratello, grazie :-)